packages/picasso.js/src/web/renderer/canvas-renderer/shapes/text.js

```diff
@@ -19,4 +19,7 @@ export default function render(t, { g, ellipsed }) {
   const bdy = baselineHeuristic(t);
 
   g.fillText(text, t.x + t.dx, t.y + t.dy + bdy);
+  if (t.stroke) {
```

Comment:
there is also the doStroke option in the second parameter.
but that also checks stroke-width. I am unsure if that is relevant for text.

Is there a use case for doing only strokeText and not fillText? (by setting fill to none)

Reply:
I can use doStroke instead, as we have both stroke and strokeWidth for the overlayText.

no there is not that I have seen where fill is none, but strokeText is needed to use stroke and strokeWidth on text.